deltachat-ios/Controller/AccountSetupController.swift

```diff
@@ -88,6 +91,7 @@ class AccountSetupController: UITableViewController, ProgressAlertHandler {
         cell.tag = tagPasswordCell
         cell.textField.tag = tagTextFieldPassword  // will be used to eventually show oAuth-Dialogue when selecting
         cell.textField.addTarget(self, action: #selector(textFieldDidChange), for: .editingChanged)
+        cell.textField.returnKeyType = advancedSectionShowing ? .next : .done
```

Comment:
i think, "done" implies that the viewController is left, this is not the case (and not wanted). also, this "big blue button" is more visible that the more important "Login" in the upper right corner, it takes too much attention imho.

therefore, i would use

```suggestion
        cell.textField.returnKeyType = advancedSectionShowing ? .next : .default
```